### PR TITLE
Main class renamed, small corrections

### PR DESCRIPTION
**Pull request by slaskawi**

### CODE REVIEW

File: src/main/java/org/jboss/pnc/buildagent/BuildAgent.java [hunk 1]
```diff
@@ -41,9 +42,9 @@
 /**
  * @author <a href="mailto:matejonnet@gmail.com">Matej Lazar</a>
  */
-public class Main {
+public class BuildAgent {
```
Comment (matejonnet) [inline src/main/java/org/jboss/pnc/buildagent/BuildAgent.java]: I prefer name Main. As a new user can quickly see where is the entry point.

File: src/main/java/org/jboss/pnc/buildagent/BuildAgent.java [hunk 2]
```diff
@@ -161,4 +179,17 @@ void notifyStatusUpdated(PtyStatusEvent statusUpdateEvent) {
     public Consumer<TtyConnection> getPtyBootstrap() {
         return ptyBootstrap;
     }
+
+    public void stop() {
+        undertowBootstrap.stop();
+        System.out.println("Server stopped");
```
Comment (matejonnet) [inline src/main/java/org/jboss/pnc/buildagent/BuildAgent.java]: Use logging.

**Comment by slaskawi** [comment]
Repushed with Nicks comments

**Comment by slaskawi** [comment]
retest this please

**Pull request merged**